plot_likert/plot_likert.py

```diff
@@ -52,6 +52,10 @@ def plot_counts(
     # Pad each row/question from the left, so that they're centered around the middle (Neutral) response
     scale_middle = len(scale) // 2
 
+    # scales labels to correct percentages
+    if plot_percentage:
+        counts = likert_percentages(counts, scale, from_count=True)
```

Comment:
One thing I wanted to flag really quickly is that there seems to be some mismatch between the named argument here (`from_count`) and the one you introduced below (`counted`).

Reply:
> One thing I wanted to flag really quickly is that there seems to be some mismatch between the named argument here (from_count) and the one you introduced below (counted).

Yes sorry about that, I changed the name from `from_count` to `counted` at the last minute because I thought it sounded a bit cleaner. 

> I think the reason this behavior exists is because plot_counts is honestly mostly an internal method, and the preferred/main method to use for plotting is plot_likert, which already does the exact conversion you're suggesting

I agree that `plot_counts` does seem to act as mostly an internal method however, it is a very useful tool to have as an external method and it is mentioned in the documentation as an option (it saved me a lot of time !). Maybe it would make more sense to incorporate `plot_counts` into `plot_likert` as an option to be enabled ? Or maybe to encapsulate the behaivour of `plot_counts` in another function that is exposed ?

Anyway thanks for taking the time to review the pull request - this library is really great !